src/components/safe-apps/AppCard.tsx

```diff
@@ -7,29 +7,50 @@ import CardContent from '@mui/material/CardContent'
 import CardHeader from '@mui/material/CardHeader'
 import IconButton from '@mui/material/IconButton'
 import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
 import { SafeAppData } from '@gnosis.pm/safe-react-gateway-sdk'
 import { SAFE_REACT_URL } from '@/config/constants'
 import useChainId from '@/hooks/useChainId'
 import ShareIcon from '@/public/images/share.svg'
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder'
+import BookmarkIcon from '@mui/icons-material/Bookmark'
+//import DeleteIcon from '@/public/images/delete.svg'
```

Comment:
Can this be removed? And then the `svg` as well?

Reply:
I see the delete button is commented out below as well. I'm guessing this is an upcoming feature?

Reply:
Yes, it was there before. It will be used for deleting custom safe apps and will be tackled shortly but in a different PR